arkane/reference.py

```diff
@@ -93,7 +93,8 @@ def __init__(self, species=None, smiles=None, adjacency_list=None, inchi=None, r
             elif inchi:
                 species = Species(inchi=inchi)
             elif adjacency_list:
-                species = Species().from_adjacency_list(adjacency_list)
+                species = Species().from_adjacency_list(adjacency_list, raise_atomtype_exception=False,
```

Comment:
Oh, this is great. Does this mean it'll now work for all species in the reference dataset?

Reply:
It should, yes, though I think there are some edge cases for raise_atomtype_exception that I need to update